### PR TITLE
refresh registry

### PR DESCRIPTION
**Pull request by aevesdocker**

### CODE REVIEW

File: content/registry/recipes/apache.md [hunk 1]
```diff
@@ -15,13 +15,13 @@ Usually, that includes enterprise setups using LDAP/AD on the backend and a SSO
...
 
 ### Solution
 
-With the method presented here, you implement basic authentication for docker engines in a reverse proxy that sits in front of your registry.
+With the method presented here, you implement basic authentication for Docker Engine in a reverse proxy that sits in front of your registry.
 
-While we use a simple htpasswd file as an example, any other apache authentication backend should be fairly easy to implement once you are done with the example.
+While we use a simple `htpasswd` file as an example, any other apache authentication backend should be fairly easy to implement once you are done with the example.
 
 We also implement push restriction (to a limited user group) for the sake of the example. Again, you should modify this to fit your mileage.
```
Comment (usha-mandya) [inline content/registry/recipes/apache.md:26]: "fit your requirements" instead of mileage?

File: content/registry/recipes/apache.md [hunk 2]
```diff
@@ -33,11 +33,9 @@ Furthermore, introducing an extra http layer in your communication pipeline adds
 
 ## Setting things up
 
-Read again [the requirements](index.md#requirements).
+1. Read again [the requirements](index.md#requirements).
```
Comment (usha-mandya) [inline content/registry/recipes/apache.md:36]: Perhaps just "Read the requirements"?

File: content/registry/recipes/apache.md [hunk 3]
```diff
@@ -194,19 +192,27 @@ EOF
...
+Now, sign in in with a "pull-only" user (using `testuser` and `testpassword`), then pull back the image:
 
-    docker login myregistrydomain.com:5043
-    docker pull myregistrydomain.com:5043/test
+  ```console
+  $ docker login myregistrydomain.com:5043
+  $ docker pull myregistrydomain.com:5043/test
+  ```
 
 Verify that the "pull-only" can NOT push:
```
Comment (usha-mandya) [inline content/registry/recipes/apache.md:214]:
```suggestion
Verify that the "pull-only" cannot push:
```

**Review by usha-mandya — COMMENTED** [review]
Made some minor suggestions. Updates look good otherwise

**Review by dvdksn — APPROVED** [review]

**Pull request merged**